src/js/components/MultiSelect/stories/DoubleColumn.js

```diff
@@ -1,45 +1,42 @@
 import React, { useState } from 'react';
 import { storiesOf } from '@storybook/react';
 
-import { Box, MultiSelect } from 'mnet-ui-base';
+import { Box, MultiSelect, Text } from 'mnet-ui-base';
 
 const options = [
-  { id: 1, label: 'Test 1' },
-  { id: 2, label: 'Test 2' },
-  { id: 3, label: 'Test 3' },
-  { id: 4, label: 'Test 4' },
-  { id: 5, label: 'Test 5' },
-  { id: 6, label: 'Test 6' },
-  { id: 7, label: 'Test 7' },
-  { id: 8, label: 'Test 8' },
-  { id: 9, label: 'Test 9' },
-  { id: 10, label: 'Test 10' },
+  { id: 1, label: 'Desktop' },
+  { id: 2, label: 'Mobile' },
+  { id: 3, label: 'Tablet' },
+  { id: 4, label: 'Television' },
+  { id: 5, label: 'Bot' },
 ];
 
 const Example = () => {
-  const [value, setValue] = useState([]);
-  const [isExcluded, setIncExc] = useState(null);
+  const [value, setValue] = useState(['Mobile', 'Tablet']);
+  const [isExcluded, setIncExc] = useState(false);
 
   return (
     <Box fill align="center" justify="start" pad="large">
       <MultiSelect
         options={options}
         value={value}
         labelKey="label"
-        valueKey={{ key: 'id', reduce: true }}
+        valueKey={{ key: 'label', reduce: true }}
         onValueChange={nextValue => setValue(nextValue)}
         layout="double-column"
-        width="medium"
+        width="large"
         height="medium"
         searchPlaceholder="Search"
         searchable
         withSelectAll
         withOptionChips
-        withUpdateCancelButtons
         withInclusionExclusion
         isExcluded={isExcluded}
         onIncExcChange={nextIncExc => setIncExc(nextIncExc)}
-        renderEmptySelected={<span>Empty</span>}
+        renderEmptySelected={<Text>No Selection</Text>}
+        isEnableOutSideClick={false}
```

Comment:
Create new story with these changed props. For other apps these may not be valid